modules/virt-schedule-supported-cpu-model-vms.adoc

```diff
@@ -5,21 +5,21 @@
 [id="virt-schedule-supported-cpu-model-vms_{context}"]
 = Scheduling virtual machines with the supported CPU model
 
-You can configure a CPU model for a virtual machine (VM) or a virtual machine instance (VMI) to schedule it on a node where its CPU model is supported.
+You can configure a CPU model for a virtual machine (VM) to schedule it on a node where its CPU model is supported.
```

Comment:
Looking at the preview, the `Setting a policy attribute and CPU feature` module in the `Scheduling virtual machines` assembly also has instances of VMI in the procedure and callout text that might be good to remove as part of this PR. The YAML `name` field needs to be changed from `myvmi` to `myvm`.